src/ui/locales/da.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Slå tilskrivning til/fra",
+    "AttributionControl.MapFeedback": "Kort feedback",
+    "FullscreenControl.Enter": "Indtast fuldskærm",
+    "FullscreenControl.Exit": "Afslut fuldskærm",
+    "GeolocateControl.FindMyLocation": "Find min placering",
+    "GeolocateControl.LocationNotAvailable": "Placering ikke tilgængelig",
+    "LogoControl.Title": "Mapbox logo",
+    "Map.Title": "Kort",
+    "NavigationControl.ResetBearing": "Nulstil lejet mod nord",
+    "NavigationControl.ZoomIn": "Zoom ind",
+    "NavigationControl.ZoomOut": "Zoome ud",
+    "ScaleControl.Feet": "ft",
+    "ScaleControl.Meters": "m",
+    "ScaleControl.Kilometers": "km",
+    "ScaleControl.Miles": "mig",
+    "ScaleControl.NauticalMiles": "nm",
```

Comment:
This is the danish unit for nautical miles, but keeping nm would also be acceptable.
```suggestion
    "ScaleControl.NauticalMiles": "sømil",
```